packages/js-dpp/test/unit/document/stateTransition/DocumetsBatchTransition/validation/state/validateDocumentsBatchTransitionStateFactory.spec.js

```diff
@@ -58,15 +58,8 @@ describe('validateDocumentsBatchTransitionStateFactory', () => {
       transitions: documentTransitions.map((t) => t.toObject()),
     }, [dataContract]);
 
-    const timeInSeconds = Math.ceil(new Date().getTime() / 1000);
-
     stateRepositoryMock = createStateRepositoryMock(this.sinonSandbox);
     stateRepositoryMock.fetchDataContract.resolves(dataContract);
-    stateRepositoryMock.fetchLatestPlatformBlockHeader.resolves({
```

Comment:
I'm not sure why this change is here exactly? 👀

Reply:
Just mentioned that we have the same lines of code and removed the unnecessary one. You can find exactly the same code on line 72